suites/test_bulk_operations.py

```diff
@@ -142,16 +142,16 @@ def published_title_add_metadata(self):
         utilities.wait(2)
         search_beta_page.add_bulk_metadata(self.driver)
         utilities.wait(2)
-        lcc.log_info("This test is failing currenly to the bug CCS-4444")
-        check_that("Alert message displayed when trying to add metadata to published title",
-            utilities.get_text(self.driver, By.CSS_SELECTOR, locators.ALERT_TITLE_CSS),
-            equal_to(constants.error_message_on_edit_metadata))
+        # lcc.log_info("This test is failing currenly to the bug CCS-4444")
```

Comment:
@pbadak Please add a comment stating why this test is disabled and the Jira reference as well

Reply:
review changes incorporated . Thanks for quick review @noopurmath3 :)